google-beta/resource_compute_disk.go

```diff
@@ -1204,10 +1233,13 @@ func resourceComputeDiskDecoder(d *schema.ResourceData, meta interface{}, res ma
 		// The raw key won't be returned, so we need to use the original.
 		transformed["rawKey"] = d.Get("disk_encryption_key.0.raw_key")
 		transformed["sha256"] = original["sha256"]
-		if v, ok := d.GetOk("disk_encryption_key_raw"); ok {
-			transformed["rawKey"] = v
+
+		if kmsKeyName, ok := original["kmsKeyName"]; ok {
```

Comment:
I'm pretty sure this is ok to have in here, but wanted to call it out as it doesn't make a ton of sense since the rest of the code for KMS key isn't in here. The flatten logic isn't looking for the KMS field so it will just be ignore.

FYI: getting the KMS feature in would be complicated because it required deprecating the convenience fields like `disk_encryption_key_raw` and the associated diff suppress in order for it to work correctly. 

fyi @rileykarson

Reply:
Oh yup this is right - we want part but not all of https://github.com/terraform-providers/terraform-provider-google-beta/pull/119

The KMS stuff should be excluded but the rest is a fix for a test failure. Both were broken @ HEAD (`2.0.0`) so I fixed them in a single PR but only half of it makes sense at `1.20.0`

time traveling code is fun

Reply:
Can you take a look at ff900686ec20f908b7da38def709dc52ae626322 and make sure it's what you imagined?

Reply:
Disk looks good to me, I'll delegate to @chrisst for RegionDisk - was KmsKeyName added separately from Disk? We want those changes in `1.20.0` if we're adding KmsKeyName.